src/pages/home/HomePage.tsx

```diff
@@ -38,7 +46,10 @@ export default class HomePage extends React.Component<IHomePageProps, {}>
     public render()
     {
         return (
-       		<QueryContainer/>
+        	<FlexCol style={{height: '100%'}}>
+				<a style={{alignSelf: 'center'}} onClick={() => this.setState({ mode: this.state.mode == 2 ? 1 : 2 })}>Switch to {this.state.mode == 2 ? 'old' : 'new'} view</a>
```

Comment:
== is just what I'm used to typing and it's what I type when it makes no difference.